### ISSUE TITLE
Activating "Delete" button present on create a new project screen functioning as back button:A11y_Developer balance_Project_Functionality

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30828]
## GitHub Tags:
#A11yTCS; #A11yFunctional; #Win32; 

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:
1. Install and open the Developer Balance app
2. TAB till Hamburger and press ENTER key.
3. TAB till Project  and Press ENTER key.
4. TAB till "Add Project" button and press ENTER key.
5. TAB till "Delete" button 
6. Observe the issue 

## Actual Result:
When we activating "Delete" button present on create a new project screen will take us on back page.

## Expected Result:
Its create a new project screen there is no user of "Delete" button so it should be remove for this page. 

## Attachment:

https://github.com/user-attachments/assets/2fbe297b-8ba7-4c21-8df8-7c6c92d0643b

### Attachment for MAC platform

https://github.com/user-attachments/assets/a95d9591-ace6-4474-9b4a-d8eb7dbe0442

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by kapilvaishna** [comment]
@jsuarezruiz Please do not close this bug. This bug should only be closed by Trusted Tester after verification.
Could you please share the latest env for all Win32 and MAC platform to verify and close this issue
